src/main/webapp/script.js

```diff
@@ -70,11 +69,33 @@ function loadGrocery() {
 
 function createGroceryElement(grocery){
     const groceryElement = document.createElement('li');
-        groceryElement.className = 'task';
+    groceryElement.className = 'task';
 
+    // changes string depending on if a member was assigned an email. 
     const titleElement = document.createElement('span');
-    titleElement.innerText = grocery;
+    const emptyString = " ";
```

Comment:
as commented above, would suggest using null and checking for === null or even !grocery.email

Then there's no need to save the constant

Reply:
Whenever I change it from " " to null my code reacts the wrong way. not sure if I can change it.

Reply:
Actually this sis work but the other comment you made earlier about changing something to null still doesn't work.

Reply:
What is the output that you get from your servlet if you go directly to the servlet URL (the URL that is being fetched by the asynchronous javascript fetch)?

This should be changeable; it's just a matter of making sure that we're making the right assumptions about how to handle the integration. There may be something about the json comparison that causes the values to look different/

Reply:
Thank you for rechecking!

Can you clarify what isn't working? Sample test cases that failed, what you tried and what happened, possible explanations all appreciated. :)